src/cljs/main/broadfcui/common/components.cljs

```diff
@@ -757,7 +764,8 @@
        (if (.-loading res)
          "Loading..."
          (let [show-counts? (:show-counts? props)
-               tag-text (.createTextNode js/document (or (.-tag res) (.-text res)))
+               ;; Insert zero-width space after underscore to allow line wrapping
+               tag-text (.createTextNode js/document (string/replace-all (or (.-tag res) (.-text res)) #"\_" "_\u200b"))
```

Comment:
we're allowing wrapping on underscores? That feels wrong.

Reply:
Otherwise some of the attribute names are monstrously long and force a horizontal scroll

Reply:
I think we'll still have the same problem if someone uses periods in a super long attribute name. Instead of special handling for underscores, how about a `word-break: break-all` CSS style?